src/main/java/com/labsynch/labseer/db/migration/postgres/V2_0_1_4__set_defaults.java

```diff
@@ -0,0 +1,497 @@
+package com.labsynch.labseer.db.migration.postgres;
```

Comment:
@brianbolt I'd broadly like a tour of how you went about merging the flyway migrations, and what the expected migration process will be